src/huggingface_hub/hf_api.py

```diff
@@ -81,27 +81,8 @@
     DiscussionWithDetails,
     deserialize_event,
 )
-from .constants import (
-    DEFAULT_ETAG_TIMEOUT,
-    DEFAULT_REQUEST_TIMEOUT,
-    DEFAULT_REVISION,
-    DISCUSSION_STATUS,
-    DISCUSSION_TYPES,
-    ENDPOINT,
-    INFERENCE_ENDPOINTS_ENDPOINT,
-    REGEX_COMMIT_OID,
-    REPO_TYPE_MODEL,
-    REPO_TYPES,
-    REPO_TYPES_MAPPING,
-    REPO_TYPES_URL_PREFIXES,
-    SAFETENSORS_INDEX_FILE,
-    SAFETENSORS_MAX_HEADER_LENGTH,
-    SAFETENSORS_SINGLE_FILE,
-    SPACES_SDK_TYPES,
-    WEBHOOK_DOMAIN_T,
-    DiscussionStatusFilter,
-    DiscussionTypeFilter,
-)
```

Comment:
@WizKnight it seems this PR broke something in the tests. There are 178 failed test with the same error: 

> FAILED ../tests/test_file_download.py::TestNormalizeEtag::test_resolve_endpoint_on_regular_file - AttributeError: <module 'huggingface_hub.hf_api' from '/home/runner/work/huggingface_hub/huggingface_hub/src/huggingface_hub/hf_api.py'> does not have the attribute 'ENDPOINT'

Makes me realize we should keep all those previously existing constants in the module even if we don't use them. You can do that by adding 

> # noqa: F401 # kept for backward compatibility

to each of them. Since `huggingface_hub.hf_api` is "public", we can't assume that no-one in the ecosystem has imported a constant from this module. It's unfortunately a bit ugly but we have to do that for backward compatibility.

Reply:
@Wauplin Yeah, it seems so. 'Cause I worked on those failed tests, I got it down to 94 failed, 189 passed, 6 skipped, 3 errors.
Though for `make quality`: mypy src
Success: no issues found in 106 source files

Reply:
All the tests passed!!! Just for ENDPOINT I reverted back to the previous import. It resolved the issue.

Reply:
Okay, I'll add backward compatibility to the imports